examples/tutorials/traits_4.0/extended_trait_change/properties.py

```diff
@@ -124,7 +127,7 @@ class LeagueModelView(ModelView):
     def _get_total_hits(self):
         """ Returns the total number of hits across all teams and players.
         """
-        return reduce(add, [reduce(add, [p.hits for p in t.players], 0)
+        return sm.reduce(add, [sm.reduce(add, [p.hits for p in t.players], 0)
```

Comment:
Gah. I'm not sure why we're so fond of `reduce`-based examples. I have a suspicion that much of this dates from before Python 2.5, when `sum` was introduced. Still, that's not something to change in this PR.